webpack/response_set.js

```diff
@@ -1,27 +1,6 @@
-const $ = require('jquery');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ResponseSetTable from './components/ResponseSetTable';
 
-exports.addResponseRow = function() {
-  const tableRowCount = $("#response-set-table tr").length - 1;
-  const newRow = `
-        <tr>
-          <td>
-            <label class="hidden" for="response_set_responses_attributes_${tableRowCount}_value">Value</label>
-            <input type="text" name="response_set[responses_attributes][${tableRowCount}][value]" id="response_set_responses_attributes_${tableRowCount}_value">
-          </td>
-          <td>
-            <label class="hidden" for="response_set_responses_attributes_${tableRowCount}_code_system">Code system</label>
-            <input type="text" name="response_set[responses_attributes][${tableRowCount}][code_system]" id="response_set_responses_attributes_${tableRowCount}_code_system">
-          </td>
-          <td>
-            <label class="hidden" for="response_set_responses_attributes_${tableRowCount}_display_name">Display name</label>
-            <input type="text" name="response_set[responses_attributes][${tableRowCount}][display_name]" id="response_set_responses_attributes_${tableRowCount}_display_name">
-          </td>
-      </tr>`;
-  const tbl = $('#response-set-table > tbody:last-child');
-  tbl.append(newRow);
-}
-
-exports.removeResponseRow = function(event) {
-  const row = $(event.target).parents("tr");
-  row.remove();
-}
+const responses = JSON.parse(document.getElementById('responses-json').innerHTML);
```

Comment:
Is this the new convention for how this sort of thing is done now?  I see the js generated in a script tag in a different file which is now being parsed for it's javascript.   Is there a downside to doing this in the initial script tag?